### ISSUE TITLE
ERROR! 'tld' is both an index level and a column label, which is ambiguous. 

### ISSUE THREAD

**Issue by ra-at-diladele-com** [diladele/websafety#1112]
UI / Traffic Monitor / Users / Top 30 Users.

```
Traceback (most recent call last):
  File "/opt/websafety/var/console/traffic/views/realtime.py", line 193, in get_context_data
    [HitsType.ALL_AVAILABLE, HitsType.ONLY_BLOCKED][blocked]
  File "/opt/websafety/var/console/_domain/traffic/users.py", line 88, in retrieve
    view = view.sort_values("tld", ascending=False)
  File "/usr/local/lib/python3.6/dist-packages/pandas/core/frame.py", line 4719, in sort_values
    k = self._get_label_or_level_values(by, axis=axis)
  File "/usr/local/lib/python3.6/dist-packages/pandas/core/generic.py", line 1702, in _get_label_or_level_values
    self._check_label_or_level_ambiguity(key, axis=axis)
  File "/usr/local/lib/python3.6/dist-packages/pandas/core/generic.py", line 1656, in _check_label_or_level_ambiguity
    raise ValueError(msg)
ValueError: 'tld' is both an index level and a column label, which is ambiguous.
```

How to fix - see https://groups.google.com/forum/#!topic/web-safety/KTYASf8tPFM

**Comment by ra-at-diladele-com** [comment]
Workaround. Open the /opt/websafety/var/console/_domain/traffic/users.py
On line 88 you will see:
```
view = view.sort_values("tld", ascending=False)
```
change it to
```
view.columns = ['hits']
view = view.sort_values("hits", ascending=False)
view.columns = ['tld']
```
Save it and restart apache/box. (I assume you know how to change python code – whitespace matter!)

 

Please tell me if it works.

Raf

**Comment by ra-at-diladele-com** [comment]
Fixed in 7.1 from https://www.diladele.com/download_next_version.html

**Comment by ra-at-diladele-com** [comment]
The traffic monitoring is completely redesigned in 8.0, closing as not relevant any more.
